src/client/components/features/ContratEngagementJeune/Rappel/FormulaireDeContact/FormulaireDeContactCEJ.test.tsx

```diff
@@ -40,32 +40,35 @@ describe('<FormulaireDeContactCEJ />', () => {
 		renderComponent();
 		// When
 		// Then
-		expect(screen.getByText('Prénom')).toBeInTheDocument();
-		expect(screen.getByText('Nom')).toBeInTheDocument();
-		expect(screen.getByText('Adresse email')).toBeInTheDocument();
-		expect(screen.getByText('Téléphone')).toBeInTheDocument();
+		expect(screen.getByLabelText('Prénom')).toBeInTheDocument();
+		expect(screen.getByLabelText('Nom')).toBeInTheDocument();
+		expect(screen.getByLabelText('Adresse email')).toBeInTheDocument();
+		expect(screen.getByLabelText('Téléphone')).toBeInTheDocument();
 		expect(screen.getByText('Age', { exact: true })).toBeInTheDocument();
-		expect(screen.getByText('Ville')).toBeInTheDocument();
+		expect(screen.getByLabelText('Ville')).toBeInTheDocument();
 		expect(screen.getByRole('button', { name: 'Envoyer la demande' })).toBeInTheDocument();
 	});
 
-	for (const label of labels.filter((l) => l !== 'Age')) {
-		it(`a un champ ${label} obligatoire`, async () => {
-			// Given
-			renderComponent();
-			// When
-			await userEvent.type(screen.getByLabelText(label), 's{backspace}');
-			// Then
-			expect(screen.getByLabelText('Nom')).toBeInvalid();
+	describe('Quand les champs sont obligatoires', () => {
+		it.each([
+			{ label: 'Prénom' },
+			{ label: 'Nom' },
+			{ label: 'Adresse email' },
+			{ label: 'Age' },
+			{ label: 'Ville' },
+		])('pour %j on retourne required', (queryParametersToTestRequired) => {
+			const result = DemandeDeContactCEJValidator.validate(queryParametersToTestRequired);
+
+			expect(result.error).toBeDefined();
 		});
-	}
+	});
```

Comment:
Ce test vérifie une fonction qui est dans un usecase (côté serveur). Or ici nous sommes dans un test de composant. Je vois que le validateur est déjà testé dans le fichier `envoyerDemandeDeContactCEJ.usecase.test.ts`, mais devrait être passé sur un it.each comme tu l'as bien fait ici